### PR TITLE
R ratio updates

### LINKED ISSUE #98: Set range for R Ratio instead of just minimum
Ideally let the user set both the min and max value on a slider instead of just setting a min value with a text box. Going to have to worry a bit about browser compatibility. 

@samussiah would it be easy to apply the approach from the recent query overview work here?

### PR DESCRIPTION
**Pull request by jwildfire**
Fix #98 #210

### CODE REVIEW

**Comment by jwildfire** [comment]
merge after #231

File: src/callbacks/onLayout/formatRRatioControl.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+            .append('button')
+            .style('padding', '0.2em 0.5em 0.2em 0.4em')
+            .style('margin-left', '0.5em')
+            .style('border-radius', '0.4em')
+            .text('Reset')
+            .on('click', function() {
+                config.r_ratio[0] = 0;
+                min_r_ratio.select('input#r_ratio_min').property('value', config.r_ratio[0]);
+                config.r_ratio[1] = config.max_r_ratio;
+                max_r_ratio.select('input#r_ratio_max').property('value', config.r_ratio[1]);
```
Comment (samussiah) [inline src/callbacks/onLayout/formatRRatioControl.js]: `max_r_ratio` gets whacked on line 24.  Think you want `min_r_ratio` here.

File: src/configuration/syncControlInputs.js [hunk 2]
```diff
@@ -83,7 +83,7 @@ export default function syncControlInputs(controlInputs, settings) {
     //drop the R Ratio control if r_ratio_filter is false
     if (!settings.r_ratio_filter) {
         controlInputs = controlInputs.filter(
-            controlInput => controlInput.label != 'Minimum R Ratio'
+            controlInput => ['r_ratio[0]', 'r_ratio[1]'].indexOf(controlInput.option) == 0
```
Comment (samussiah) [inline src/configuration/syncControlInputs.js]: `< 0`, not `== 0`

File: src/callbacks/onLayout/formatRRatioControl.js (hunk 1 above)
Comment (jwildfire) [reply src/callbacks/onLayout/formatRRatioControl.js]: I think it's ok as is ... Moving the input for max to the wrap for min, and then deleting the wrap for max.

File: src/configuration/syncControlInputs.js (hunk 2 above)
Comment (jwildfire) [reply src/configuration/syncControlInputs.js]: doh. good catch. thanks.

**Comment by jwildfire** [comment]
Cleaned up issues from code review. Merging now.

File: src/callbacks/onLayout/formatRRatioControl.js (hunk 1 above)
Comment (samussiah) [reply src/callbacks/onLayout/formatRRatioControl.js]: The reset button doesn't reset the max r-ratio input.  The element `max_r_ratio` references doesn't contain `input#r_ratio_max` anymore:

min-max-animation

Comment (jwildfire) [reply src/callbacks/onLayout/formatRRatioControl.js]: well crap.

**Pull request merged**